util/build_docs.py

```diff
@@ -161,6 +162,17 @@ def generate_selfdocs():
                 fout.write(tlgen.selfdoc(heading=3, cmd='tlgen.py --doc'))
 
 
+def is_hugo_extended():
+    args = [
+        "hugo",
+        "version"
+    ]
+    process = subprocess.run(args, text=True, stdout=subprocess.PIPE,
+                             check=True, cwd=str(SRCTREE_TOP))
+
+    return bool(re.search("v\d+\.\d+\.\d+-.*/extended", process.stdout))
```

Comment:
This should work, but I would go less fancy here, given that the version string doesn't seem to be particularly stable.
`return process.stdout.find("extended")`

If you keep the regex, please add a comment with the of "hugo version" to make the regex easier to understand.

Reply:
My thinking was that its potentially more likely that they could add some other "extended", which then could result in an incorrect behaviour. This way seems more robust as long as the version portion of the string stays the same. Probably fine either way really.